include/mellon/impl/extensions.h

```diff
@@ -60,6 +62,7 @@ struct FUTURES_EMPTY_BASE future_type_based_extensions<expect::expected<T>, Fut,
     return std::move(self()).and_capture(std::forward<F>(f));
   }*/
 
+  // mpoeter - documentation missing
```

Comment:
Would you like to provide it? Otherwise I will do...

Reply:
I thought about that, but I am not 100% sure what these functions do, so I would prefer if you could do it.